hack/lib/golang.sh

```diff
@@ -24,6 +24,7 @@ readonly KUBE_GOPATH="${KUBE_OUTPUT}/go"
 # If you update this list, please also update build/release-tars/BUILD.
 kube::golang::server_targets() {
   local targets=(
+    cmd/gke-certificates-controller
```

Comment:
Ah. Is it more appropriate to add the target to `KUBE_ALL_TARGETS` instead? I don't actually need it in the tar, I just want to make sure that it gets built by default when anyone runs `make` so we'll prevent anyone from breaking the code.